package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaba",
-  "version": "1.0.0-beta.43",
+  "version": "1.0.0-beta.44",
```

Comment:
This doesn't need to be manually bumped, it will be automatically bumped during a release.